tests/test_plugin_detection.py

```diff
@@ -80,7 +78,8 @@ def test_detect_plugin(self):
         for linker in man.media_linkers:
```

Comment:
We can't comment on non modified lines, so commenting here: You should modify the `setUp` method to remove the dependency on `pkg_resources`. You can see what I previously did in https://github.com/AcademySoftwareFoundation/OpenTimelineIO/compare/main...JeanChristopheMorinPerso:OpenTimelineIO:importlib?expand=1#diff-3d0c116209043c6578f8bb39f8d5fc32f70fad767498ef5a2e027af4396b5b71.

Reply:
Good call.  Borrowing your code...

Reply:
Thank you for providing your branch, @JeanChristopheMorinPerso - it was super helpful.

Reply:
It's a pleasure!